app.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+class App
+  attr_accessor :books, :peoples, :rentals
+
+  def initialize
+    @books = []
+    @peoples = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'No books available'
+    else
+      @books.each { |book| puts "#{book.title} by #{book.author}" }
+    end
+  end
+
+  def list_people
+    if @peoples.empty?
+      puts 'No people available'
+    else
+      @peoples.each { |person| puts "ID: #{person.id} , Name: #{person.name}, Age: #{person.age}" }
+    end
+  end
+
+  def create_student(age, has_parent_permission, name)
+    student = Student.new(age, parent_permission: has_parent_permission, name: name)
+    @peoples << student
+  end
+
```

Comment:
- [ ] Good job creating a basic UI for your app, However , I once I wanted to create Student , He could not be saved in the `students` variable 👍 ,even though no exception was thrown 😄 . Kindly ensure that I can create a student in your app 👍